openquake/hazardlib/gsim/kuehn_2020.py

```diff
@@ -737,15 +738,12 @@ def __init__(self, region="GLO", m_b=None, m9_basin_term=None,
         self.region = region
 
         # For some regions a basin depth term is defined
-        if self.region in ("CAS", "JPN"):
-            # If region is CAS or JPN then the GMPE needs Z2.5
-            self.REQUIRES_SITES_PARAMETERS = \
-                 self.REQUIRES_SITES_PARAMETERS.union({"z2pt5", })
+        if self.region in ("CAS", "JPN", "Sea"):
```

Comment:
It would be better to use an uppercase convention, i.e. SEA instead of Sea.

Reply:
I've just updated this now thanks.